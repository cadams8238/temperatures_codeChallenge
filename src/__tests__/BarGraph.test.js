import React from "react";
import BarGraph from "../components/barGraph";
import { shallow } from "enzyme";
import renderer from "react-test-renderer";

describe("<BarGraph />", () => {
	const data = [
		{
			date: "1/31",
			"Max Temp": 45,
			"Min Temp": 28
		},
		{
			date: "2/1",
			"Max Temp": 45,
			"Min Temp": 28
		}
	];

	it("Should render without crashing", () => {
		shallow(<BarGraph graphData={data} />);
	});

	it("Snapshot test", () => {
		const tree = renderer.create(<BarGraph graphData={data} />).toJSON();
		expect(tree).toMatchSnapshot();
	});
});
