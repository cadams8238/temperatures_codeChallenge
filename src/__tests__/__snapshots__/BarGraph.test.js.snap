// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BarGraph /> Snapshot test 1`] = `
<div
  className={undefined}
>
  <div
    style={
      Object {
        "height": "100%",
        "width": "100%",
      }
    }
  />
</div>
`;
